README.md

```diff
@@ -305,9 +402,9 @@ While we each bring a different perspective and set of expertise to customer int
 
         * Ask if it's a good time
 
-    * Need you in the next hour: Google Hangouts
+    * Need you in the next hour: Podio chat
 
-    * Need you within the day or longer: email
+    * Need you within the day or longer: Podio chat, update, or task
```

Comment:
Yes, only because it is the best way to communicate privately (1x1) even if you need someone within a day. I'll clarify that it's only for the private case.